modules/openapi-generator/src/test/java/org/openapitools/codegen/java/helidon/functional/AbstractFunctionalHelidonClientTest.java

```diff
@@ -26,7 +26,7 @@
 import static org.hamcrest.CoreMatchers.not;
 import static org.hamcrest.MatcherAssert.assertThat;
 
-public class FunctionalHelidonClientBase extends FunctionalBase {
+abstract class AbstractFunctionalHelidonClientTest extends AbstractFunctionalHelidonTest {
```

Comment:
These functional Helidon tests did not follow a standard naming convention and so they were not being picked up by Gradle. I've simply renamed the classes here.